common.blocks/menu/menu.bh.js

```diff
@@ -31,6 +37,8 @@ module.exports = function(bh) {
                     var i = 0, itemOrGroup;
                     while(itemOrGroup = content[i++]) {
                         if(itemOrGroup.block === 'menu-item') {
+                            itemOrGroup.attrs = (itemOrGroup.attrs || {});
+                            itemOrGroup.attrs.id = itemOrGroup.attrs.id || ctx.generateId(); // Support a11y
```

Comment:
Oh, sh.. But this mean that `generateId()` [from BEMHTML][1] and `bh` works in different ways?

It looks like, `bh`'s `ctx.generateId()` is `this.identify()` from BEMHTML :(

[1]: https://github.com/bem/bem-core/blob/6ce978f653cd84a7a490e654d6c6dbf378cb7546/common.blocks/i-bem/i-bem.bemhtml#L241-L243